runelite-client/src/main/java/net/runelite/client/plugins/wintertodt/WintertodtPlugin.java

```diff
@@ -219,6 +220,19 @@ public void onVarbitChanged(VarbitChanged varbitChanged)
 		}
 	}
 
+	@Subscribe
+	public void onGameStateChanged(GameStateChanged event)
+	{
+		switch (event.getGameState())
+		{
+			case HOPPING:
+			case LOGGING_IN:
+				previousTimerValue = 0;
+				client.setVarbit(Varbits.WINTERTODT_TIMER, 0);
```

Comment:
We really shouldn't be setting the var ourselves. A better way to handle this would be to add a check in the varbit change subscriber to prevent writing to the field when the game state is `LOADING`.